src/style.css

```
@@ -1,3 +1,62 @@
 body {
   margin: 0;
 }
+
+img {
+  width: 40px;
+  height: 40px;
+}
+
+.link__logo {
+  margin-left: 50px;
+}
+
+header {
+  font-family: Roboto, sans-serif;
+  max-width: 1200px;
+  min-width: 1024px;
+  width: 100%;
+  height: 60px;
+  background: #fff;
+  box-shadow: 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__list {
+  height: 15px;
+  list-style: none;
+  display: flex;
+  margin-right: 60px;
+}
+
+.nav__item {
```

first item shouldn't have margin
```suggestion
.nav__item:not(:first-child) {
```